massa-grpc/src/stream/tx_throughput.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2023 MASSA LABS <info@massa.net>
+
+use crate::{error::GrpcError, service::MassaGrpcService};
+use futures_util::StreamExt;
+use massa_proto::massa::api::v1 as grpc;
+use std::pin::Pin;
+use std::time::Duration;
+use tokio::{select, time};
+use tonic::codegen::futures_core;
+use tracing::log::error;
+
+// Throughput interval in seconds
+const THROUGHPUT_INTERVAL: u64 = 10;
```

Comment:
Should this not be in a config file?

Reply:
maybe rename it to `DEFAULT_THROUGHPUT_INTERVAL` ?

Reply:
If so, then you could change the name to DEFAULT_THROUGHPUT_INTERVAL, and add a comment to explain? You know more about this than me :)

Reply:
it's ok for you ? https://github.com/massalabs/massa/pull/3443/commits/06ce7872f3963cedd8963925dde1f1dbda8f2dc8

Reply:
Yes, all good